_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -185,713 +118,31 @@ Topics:
-#  - Name: Understanding identity provider configuration
-#    File: understanding-identity-provider
-# - Name: Configuring identity providers
-#   File: sd-configuring-identity-providers
-#  - Name: Configuring identity providers
-#    Dir: identity_providers
-#    Topics:
-#    - Name: Configuring an htpasswd identity provider
-#      File: configuring-htpasswd-identity-provider
-#    - Name: Configuring a Keystone identity provider
-#      File: configuring-keystone-identity-provider
-#    - Name: Configuring an LDAP identity provider
-#      File: configuring-ldap-identity-provider
-#    - Name: Configuring a basic authentication identity provider
-#      File: configuring-basic-authentication-identity-provider
-#    - Name: Configuring a request header identity provider
-#      File: configuring-request-header-identity-provider
-#    - Name: Configuring a GitHub or GitHub Enterprise identity provider
-#      File: configuring-github-identity-provider
-#    - Name: Configuring a GitLab identity provider
-#      File: configuring-gitlab-identity-provider
-#    - Name: Configuring a Google identity provider
-#      File: configuring-google-identity-provider
-#    - Name: Configuring an OpenID Connect identity provider
-#      File: configuring-oidc-identity-provider
-# - Name: Using RBAC to define and apply permissions
-#   File: using-rbac
-#  - Name: Removing the kubeadmin user
-#    File: remove-kubeadmin
-# - Name: Configuring LDAP failover
-#   File: configuring-ldap-failover
-# - Name: Understanding and creating service accounts
-#   File: understanding-and-creating-service-accounts
-# - Name: Using service accounts in applications
-#   File: using-service-accounts-in-applications
-# - Name: Using a service account as an OAuth client
-#   File: using-service-accounts-as-oauth-client
-# - Name: Assuming an AWS IAM role for a service account
-#   File: assuming-an-aws-iam-role-for-a-service-account
-# - Name: Scoping tokens
-#   File: tokens-scoping
-# - Name: Using bound service account tokens
-#   File: bound-service-account-tokens
-# - Name: Managing security context constraints
-#   File: managing-security-context-constraints
-# - Name: Understanding and managing pod security admission
-#   File: understanding-and-managing-pod-security-admission
-#  - Name: Impersonating the system:admin user
-#    File: impersonating-system-admin
-# - Name: Syncing LDAP groups
-#   File: ldap-syncing
-#  - Name: Managing cloud provider credentials
-#    Dir: managing_cloud_provider_credentials
-#    Topics:
-#    - Name: About the Cloud Credential Operator
-#      File: about-cloud-credential-operator
-#    - Name: Mint mode
-#      File: cco-mode-mint
-#    - Name: Passthrough mode
-#      File: cco-mode-passthrough
-#    - Name: Manual mode with long-term credentials for components
-#      File: cco-mode-manual
-#    - Name: Manual mode with short-term credentials for components
-#      File: cco-short-term-creds
----
```

Comment:
This marks the beginning of a book, but all the lines from here until the next stanza start (`---`) are commented out, so build script sees this as a book missing its `Name:` directive etc.